nuitka/nodes/BuiltinAnyNodes.py

```diff
@@ -29,14 +29,12 @@
 class ExpressionBuiltinAny(ExpressionBuiltinSingleArgBase):
     """ Builtin Any Node class.
 
-    Parameter
-    ---------
-    ExpressionBase: 'any' expression
-
-    Returns
-    -------
-    Bool: True if at least one element of an iterable is true
-          else return False
+    Args:
+        ExpressionBase: 'any' expression
```

Comment:
According to this: http://nuitka.net/doc/developer-manual.html#api-documentation-and-guidelines

It should be 'any - ...'

Reply:
Sorry, I can't find it in the docs. Do you it should be 

```
Args:
    ExpressionBase: 'any - expression'
```